tools/libSwiftScan/CMakeLists.txt

```diff
@@ -46,20 +46,20 @@ endif()
 
 if(SWIFT_BUILD_SWIFT_SYNTAX)
   if(SWIFT_HOST_VARIANT_SDK IN_LIST SWIFT_DARWIN_PLATFORMS)
-    # Ensure that we can find the host shared libraries.
+    # Ensure that we can find the shared swift-syntax libraries.
     set_property(
       TARGET libSwiftScan
-      APPEND PROPERTY INSTALL_RPATH "@loader_path/swift/host")
+      APPEND PROPERTY INSTALL_RPATH "@loader_path/swift/host/compiler")
     set_property(
       TARGET libSwiftScan
-      APPEND PROPERTY INSTALL_RPATH "@loader_path/../host")
+      APPEND PROPERTY INSTALL_RPATH "@loader_path/compiler")
```

Comment:
https://github.com/apple/swift/commit/1e93fb34f821941ae533a46e17af9b8d0eb44684
Was there a reason for `@loader_path/../host` instead of `@loader_path`?

Reply:
Yes, it's symlinked in `lib/swift/linux` etc.